cloud-platform/env-create.md

```diff
@@ -0,0 +1,232 @@
+
+This Rolebinding resource references the ClusterRole to provide ClusterRole admin access to a set of users which is defined under subjects. In this case, the `$yourTeam` github group will have admin access to any resources within the namespace "myapp dev".
+
+`$servicename-$env-admin-role.yaml`
+
+Example of yaml above, i will explain parts that need to be changed.
+
+```
+kind: 
+apiVersion:
+metadata:
+     name:
+     namespace: Define your namespace as created in namespace.yaml, this is where its mapped.
+subjects:
+     kind: 
+     name: This is where you specify your Github team, as format shown above "github:$yourTeam"
+     apiGroup:
+roleRef: 
+     kind: 
+     name:
+     apiGroup:
+```
+
+[https://kubernetes.io/docs/admin/authorization/rbac/](https://kubernetes.io/docs/admin/authorization/rbac/)
+
+```
+within our $servicename-$env directory, you will need to create the two files shown above. You can use any file editor of your choice (nano, vim).
+
+$ cd myapp-dev/       								# Make sure you are in your service directory.
+$ vi namespace.yaml   								#create the namespace.yaml and use guide above to help structure file.
+$ vi myapp-dev-admin-role.yaml 						#once completed do the same for admin-role file.
+$ ls                            					#list files in $Servicename-$dev directory, to see both files have been saved.
+myapp-dev-admin-role.yaml	namespace.yaml   		#output after running list command.
+
+##Remember these are Yaml files, so make sure syntax is correct. If syntax is incorrect then file may not build properly, you can use a online validator.
+#https://codebeautify.org/yaml-validator
+
+```
+
+4) After both files are created. You are ready to upload your branch to our github repo. You will need to add the files, commit them and then set a new upstream branch.
+
+```
+# Add all new files from your current directory to staging.
+$ git add
+
+# Can run the command git status to check what changes are to be commited.
+$ git status
+
+# output will show files that are to be commited in green.
+
+Changes to be committed:
+  (use "git reset HEAD <file>..." to unstage)
+
+	new file:   namespaces/myapp-dev/myapp-dev-admin-role.yaml
+	new file:   namespaces/myapp-dev/namespace.yaml
+
+#once your happy with the files to be uploaded, you can now run git commit to commit the files and -m parameter to add message to your commit.
+$ git commit -m "adding namespace.yaml and admin-roles for myapp to create dev environment"
+
+#Should get output reflecting files added.
+
+2 files changed, 0 insertions(+), 0 deletions(-)
+ create mode 100644 namespaces/myapp-dev/myapp-dev-admin-role.yaml
+ create mode 100644 namespaces/myapp-dev/namespace.yam
+
+#Now you are ready to upload your branch to the repo. myapp being your branch name.
+git push --set-upstream origin myapp
```

Comment:
This means that we need to give everyone commit access to the repo - is that what we want, or should we have users fork the repo and issue a PR from there instead - @kalbir ?